src/ConfigPostProcessor.php

```diff
@@ -14,13 +14,17 @@
 use function array_pop;
 use function array_push;
 use function count;
+use function in_array;
 use function is_array;
 use function is_callable;
 use function is_int;
 use function is_string;
 
 class ConfigPostProcessor
 {
+    /** @var array<string> */
+    private static $SERVICE_MANAGER_KEYS_OF_INTEREST = ['aliases', 'invokables', 'factories', 'services'];
```

Comment:
I prefer using constant here, we can mark it as internal, as it cannot be private (because of PHP 5.6).

Also, I would change it to:
```php
/**
 * @internal
 */
const SERVICE_MANAGER_KEYS_OF_INTEREST = [
    'aliases' => true,
    'factories' => true,
    'invokables' => true,
    'services' => true,
];
```

we can skip `@var` with type for constant I believe.

Using this format simplify the usage latter - we don't need to use `array_filp` and instead of `in_array` we can use `isset`.

Reply:
By making it a constant, the codestyle sniff is still complying with a warning. 
Not sure if thats still the case for `@internal` but thats why I've changed it to a private static property tho.

Reply:
Check out my commit: https://github.com/boesing/laminas-zendframework-bridge/commit/3dc51591681f6da7e41a1b738ca0acbd6aa8308d

I've excluded one PSR-12 rule, as we still on PHP 5.6 here.